django/urls/converters.py

```diff
@@ -53,6 +56,14 @@ class PathConverter(StringConverter):
 
 
 def register_converter(converter, type_name):
+    if REGISTERED_CONVERTERS.get(type_name) is not None:
+        # Removed in Django60Warning: when the deprecation end, replace with
+        # raise InvalidNameConverter("converter %s already exists" % type_name)
+        warnings.warn(
+            "Using the same converter name is deprecated",
+            RemovedInDjango60Warning,
+            stacklevel=2,
+        )
```

Comment:
Doc changes are missing, check out [docs](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-code/submitting-patches/#deprecating-a-feature) for more details. There are also [unanswered questions](https://code.djangoproject.com/ticket/35090#comment:8) from Natalia about the desired behavior.